xamarin/Samples/Showcase-Android/DDLFormActivity.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using Com.Liferay.Mobile.Screens.Ddl.Form;
+using Com.Liferay.Mobile.Screens.Ddl.Model;
+using Java.Lang;
+using Org.Json;
+
+namespace ShowcaseAndroid
+{
+    [Activity]
+    public class DDLFormActivity : Activity, IDDLFormListener
+    {
+
+        private DDLFormScreenlet ddlFormScreenlet;
+
+        protected override void OnCreate(Bundle savedInstanceState)
+        {
+            base.OnCreate(savedInstanceState);
+            SetContentView(Resource.Layout.DDLFormView);
+
+            ddlFormScreenlet = (DDLFormScreenlet)FindViewById(Resource.Id.ddl_form_screenlet);
+            ddlFormScreenlet.Listener = this;
+        }
+
+        protected override void OnResume()
+        {
+            base.OnResume();
+            ddlFormScreenlet.Load();
+        }
+
+        public void Error(Java.Lang.Exception p0, string p1)
```

Comment:
Can you log something? if someone uses the app and doesnt change the default values or whatever it's going to hard to find it